lib/src/components/Surfaces/Accordion/AccordionSummary.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { ChevronDown } from "lucide-react";
+import { AccordionProps } from "./Accordion.types";
+import { twMerge } from "tailwind-merge";
+export const AccordionSummary = <C extends React.ElementType = "h3">({
+  children,
+  expanded,
+  handelChange,
+  disabled,
+  expandIcon = <ChevronDown color="#71717A" size={18} />,
```

Comment:
Get rid of hard coded hex colors
```suggestion
  expandIcon = <ChevronDown className="stroke-muted-foreground" size={18} />,
```

Reply:
Done